[package] ffmpeg/*: options flags are not correctly passed to ffmpeg build.

### Description

The ffmpeg build system uses the same flags multiple times to enable different options for categories of options, for example you might pass the `--enable-encoder=` flag multiple times `--enable-encoder=gif --enable-encoder=mp3`. These are supported in the conanfile using lists, such as `ffmpeg/*:enable_encoders=gif,mp3` and these used to be correctly pass to the ffmpeg build as multiple `--enable-encoder` etc. flags.

Using Conan 2.x (and possibly some more recent version of 1.x, I haven't tested them) these "duplicate" flags are now removed from the list of flags passed to the ffmpeg build, only the last in the list is passed through. 

Debugging this, it seems that the `AutotoolsToolchain` method `update_configure_args` is stripping these for some reason. An easy workaround for this is to append the flags after this call instead of before, this seems to work fine here without any unexpected side effects, so I will open a PR to address this.

### Package and Environment Details

* Package Name/Version: **ffmpeg/***
* Operating System+version: **Linux Manjaro**
* Compiler+version: **GCC 12.2.1**
* Conan version: **conan 2.0.4**
* Python version: **Python 3.10.9**


### Conan profile

[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=12
os=Linux


### Steps to reproduce

conan create . --version 5.1 --build=ffmpeg -o ffmpeg/\*:disable_everything=True -o ffmpeg/\*:enable_encoders="ac3, gif" -o ffmpeg/\*:enable_filters="concat" -o ffmpeg/\*:with_pulse="False" -o ffmpeg/\*:with_vaapi="False" -o ffmpeg/\*:with_vdpau="False" -o ffmpeg/\*:with_vulkan="False" -o ffmpeg/\*:with_xcb="False" -o ffmpeg/\*:with_libalsa="False" -o ffmpeg/\*:with_programs="False" -o ffmpeg/\*:with_ssl="False" --build=missing

### Logs

<details><summary>Click to expand log</summary>

```
======== Exporting recipe to the cache ========
ffmpeg/5.1: Exporting package recipe: /home/si/Projects/conan-center-index/recipes/ffmpeg/all/conanfile.py
ffmpeg/5.1: exports: File 'conandata.yml' found. Exporting it...
ffmpeg/5.1: Calling export_sources()
ffmpeg/5.1: Copied 1 '.py' file: conanfile.py
ffmpeg/5.1: Copied 1 '.yml' file: conandata.yml
ffmpeg/5.1: Copied 1 '.patch' file: 5.0-0001-fix-hwcontext_vulkan.patch
ffmpeg/5.1: Exported to cache folder: /home/si/.conan2/p/ffmpe2613385c510e9/e
ffmpeg/5.1: Exported: ffmpeg/5.1#dff19df371919871b224b2c3a8c279e5 (2023-04-20 14:36:36 UTC)

======== Input profiles ========
Profile host:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=12
os=Linux
[options]
ffmpeg/*:disable_everything=True
ffmpeg/*:enable_encoders=ac3, gif
ffmpeg/*:enable_filters=concat
ffmpeg/*:with_libalsa=False
ffmpeg/*:with_programs=False
ffmpeg/*:with_pulse=False
ffmpeg/*:with_ssl=False
ffmpeg/*:with_vaapi=False
ffmpeg/*:with_vdpau=False
ffmpeg/*:with_vulkan=False
ffmpeg/*:with_xcb=False

Profile build:
[settings]
arch=x86_64
build_type=Release
compiler=gcc
compiler.cppstd=gnu17
compiler.libcxx=libstdc++11
compiler.version=12
os=Linux


======== Computing dependency graph ========
Graph root
    cli
Requirements
    brotli/1.0.9#ffbecf553112bc3e25a08c91c28b8b68 - Cache
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d - Cache
    ffmpeg/5.1#dff19df371919871b224b2c3a8c279e5 - Cache
    freetype/2.13.0#a312c096b314c6cfd6b43e290c88928a - Cache
    libfdk_aac/2.0.2#c9eb99853a2067c754bb1a826044d6eb - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335 - Cache
    libmp3lame/3.100#44b12d19316eb2b223d98d3e75dae438 - Cache
    libpng/1.6.39#a4953955f060811c79abff9fe986b6fe - Cache
    libvpx/1.11.0#6fbd55b629d52b82db5786b0dd84a48b - Cache
    libwebp/1.3.0#d92654dbce1f8cb8ae45d5388f86fcfe - Cache
    libx264/cci.20220602#1847939486c0bfc5141898c469f9fff4 - Cache
    libx265/3.4#fb1af2d1f0f1c8aa25510237f1d86c39 - Cache
    ogg/1.3.5#062626875f5c8c59f069f76f148098ef - Cache
    openh264/2.3.1#e61a2e588611d5f611fdca7cbc4947b2 - Cache
    openjpeg/2.5.0#e3684ea8ab1cfc6707a461b0b2d0bb9c - Cache
    opus/1.3.1#6938146de9da3b6817c85b6d522e8e4d - Cache
    vorbis/1.3.7#d936fd8c850f0c7c4021e24e55e6352c - Cache
    xz_utils/5.4.0#186e94237b66f785e4ea095f2f621a56 - Cache
    zlib/1.2.13#13c96f538b52e1600c40b88994de240f - Cache
Build requirements
    gnu-config/cci.20210814#15c3bf7dfdb743977b84d0321534ad90 - Cache
    meson/1.0.0#15586c0ac6f682805875ef903dbe7ee2 - Cache
    nasm/2.15.05#799d63b1672a337584b09635b0f22fc1 - Cache
    ninja/1.11.1#a2f0b832705907016f336839f96963f8 - Cache
    pkgconf/1.9.3#a920b5c7f8d04f22b9fe03db91a864dd - Cache
    yasm/1.3.0#fb800a15413dca19bfaef9e4b5d50694 - Cache

======== Computing necessary packages ========
Requirements
    brotli/1.0.9#ffbecf553112bc3e25a08c91c28b8b68:18545dd496007772cd15d1f792fd745d3faf459e#4c5b1814d03a5ca2661a94bdcc12b0f8 - Cache
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d:4530633a3406a354d9c70a71124250144067de22#75e696190aa466c071d9e211bca442f7 - Cache
    ffmpeg/5.1#dff19df371919871b224b2c3a8c279e5:6ece4764455be2a75c6c9523ed82c4720829412e - Build
    freetype/2.13.0#a312c096b314c6cfd6b43e290c88928a:a7bd0acd641d011580e2dd3e4cb52f030ef01db4#49f4dc389f7b2639339c61edcf7019c0 - Cache
    libfdk_aac/2.0.2#c9eb99853a2067c754bb1a826044d6eb:2b3e00e93be912c4468bf5911338440f07c9b5ac#a6b721688ee50656ac543612b44836c8 - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#a82f7b8bf96b3aa38bdcfdc9ba5d1c75 - Cache
    libmp3lame/3.100#44b12d19316eb2b223d98d3e75dae438:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#39ef5ed10ac7f73501e2cc4e5169e718 - Cache
    libpng/1.6.39#a4953955f060811c79abff9fe986b6fe:f98ded23a1dcda1844efab049c7002f85ec1e8d1#1009087d2848fed733f2ecdabe8a7594 - Cache
    libvpx/1.11.0#6fbd55b629d52b82db5786b0dd84a48b:80f2cbb9829b9c7aaebc9f3dec78fb86ac4ff2f3#9fccc154fa4cf19da88a11a3af9afbdd - Cache
    libwebp/1.3.0#d92654dbce1f8cb8ae45d5388f86fcfe:e45efe1f2529ffe2564429d57eafb384b5ecc715#9d95215a5c91c1c271bcedfa8297acfc - Cache
    libx264/cci.20220602#1847939486c0bfc5141898c469f9fff4:3fb4c14de6c821fa43bf9d290f24719bf569ba18#cb5033d96089e1ee6460bd095fad9d48 - Cache
    libx265/3.4#fb1af2d1f0f1c8aa25510237f1d86c39:81185bfb3175decd99a7f82a317c0b5e2fc19fda#ff070ebcf5dd30819411d35021dd3b0f - Cache
    ogg/1.3.5#062626875f5c8c59f069f76f148098ef:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#730c080a224a0fd767ed0d6a54d4995b - Cache
    openh264/2.3.1#e61a2e588611d5f611fdca7cbc4947b2:2b3e00e93be912c4468bf5911338440f07c9b5ac#94e64a3dde81fc60af7273e02dcb54dc - Cache
    openjpeg/2.5.0#e3684ea8ab1cfc6707a461b0b2d0bb9c:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#802cbddfb0ab82327ba4c997a0d53b69 - Cache
    opus/1.3.1#6938146de9da3b6817c85b6d522e8e4d:5cd6a71e35ad633a50574774f9c05abbde0f0b34#e12e619c32edbcb2962fc0149b7acad0 - Cache
    vorbis/1.3.7#d936fd8c850f0c7c4021e24e55e6352c:15c679a1f3354e98b9583bcadddca4d34ae2f0b9#000c8f11c485fc0d0e6296308bb629cb - Cache
    xz_utils/5.4.0#186e94237b66f785e4ea095f2f621a56:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#c54662157e9d9abdcbd569f25c130c8e - Cache
    zlib/1.2.13#13c96f538b52e1600c40b88994de240f:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#c0a86acefcee55d678689521d3ea23ea - Cache
Build requirements
    gnu-config/cci.20210814#15c3bf7dfdb743977b84d0321534ad90:da39a3ee5e6b4b0d3255bfef95601890afd80709#22618e30bd9e326eb95e824dc90cc860 - Skip
    meson/1.0.0#15586c0ac6f682805875ef903dbe7ee2:da39a3ee5e6b4b0d3255bfef95601890afd80709#5c8fd51fc33f12e26519674d99afd0e5 - Skip
    nasm/2.15.05#799d63b1672a337584b09635b0f22fc1:3593751651824fb813502c69c971267624ced41a#6fea6fbfb5f1ce0afa9745dfce12485d - Skip
    ninja/1.11.1#a2f0b832705907016f336839f96963f8:3593751651824fb813502c69c971267624ced41a#a53f170b60a46aef75ead8658bdeae05 - Skip
    pkgconf/1.9.3#a920b5c7f8d04f22b9fe03db91a864dd:c0b621fd4b3199fe05075171573398833dba85f4#a231c33e360d8be4e90467b36b88f027 - Cache
    yasm/1.3.0#fb800a15413dca19bfaef9e4b5d50694:3593751651824fb813502c69c971267624ced41a#de209941eeb4029e79a7a33bfefe9a98 - Cache

======== Installing packages ========
brotli/1.0.9: Already installed! (1 of 21)
bzip2/1.0.8: Already installed! (2 of 21)
libfdk_aac/2.0.2: Already installed! (3 of 21)
libiconv/1.17: Already installed! (4 of 21)
libwebp/1.3.0: Already installed! (5 of 21)
ogg/1.3.5: Already installed! (6 of 21)
openjpeg/2.5.0: Already installed! (7 of 21)
opus/1.3.1: Already installed! (8 of 21)
xz_utils/5.4.0: Already installed! (9 of 21)
yasm/1.3.0: Already installed! (10 of 21)
yasm/1.3.0: Appending PATH environment variable: /home/si/.conan2/p/yasm6a26ef75fa025/p/bin
zlib/1.2.13: Already installed! (11 of 21)
libmp3lame/3.100: Already installed! (12 of 21)
libvpx/1.11.0: Already installed! (13 of 21)
libx264/cci.20220602: Already installed! (14 of 21)
libx265/3.4: Already installed! (15 of 21)
openh264/2.3.1: Already installed! (16 of 21)
pkgconf/1.9.3: Already installed! (17 of 21)
pkgconf/1.9.3: Appending PATH env var: /home/si/.conan2/p/pkgcod729a37c3fef7/p/bin
pkgconf/1.9.3: Setting PKG_CONFIG env var: /home/si/.conan2/p/pkgcod729a37c3fef7/p/bin/pkgconf
pkgconf/1.9.3: WARN: The use of 'unix_path_legacy_compat' is deprecated in Conan 2.0 and does not perform path conversions. This is retained for compatibility with Conan 1.x and will be removed in a future version.
pkgconf/1.9.3: Appending AUTOMAKE_CONAN_INCLUDES env var: /home/si/.conan2/p/pkgcod729a37c3fef7/p/bin/aclocal
libpng/1.6.39: Already installed! (18 of 21)
vorbis/1.3.7: Already installed! (19 of 21)
freetype/2.13.0: Already installed! (20 of 21)
ffmpeg/5.1: Calling source() in /home/si/.conan2/p/ffmpe2613385c510e9/s/src
Downloading ffmpeg-5.1.tar.bz2

-------- Installing package ffmpeg/5.1 (21 of 21) --------
ffmpeg/5.1: Building from source
ffmpeg/5.1: Package ffmpeg/5.1:6ece4764455be2a75c6c9523ed82c4720829412e
ffmpeg/5.1: Copying sources to build folder
ffmpeg/5.1: Building your package in /home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b
ffmpeg/5.1: Calling generate()
ffmpeg/5.1: Generators folder: /home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/build-release/conan
ffmpeg/5.1: Generating aggregated env files
ffmpeg/5.1: Generated aggregated env files: ['conanbuild.sh']
ffmpeg/5.1: Calling build()
ffmpeg/5.1: Apply patch (portability): Compatibility with vulkan >= 1.3.239
ffmpeg/5.1: Calling:
 > "/home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/configure" --disable-shared --enable-static --prefix=/ '--bindir=${prefix}/bin' '--libdir=${prefix}/lib' --pkg-config-flags=--static --disable-doc --disable-cross-compile --enable-asm --enable-pic --enable-avdevice --enable-avcodec --enable-avformat --enable-swresample --enable-swscale --enable-postproc --enable-avfilter --enable-bzlib --enable-zlib --enable-lzma --enable-iconv --enable-libopenjpeg --enable-libopenh264 --enable-libvorbis --enable-libopus --disable-libzmq --disable-sdl2 --enable-libx264 --enable-libx265 --enable-libvpx --enable-libmp3lame --enable-libfdk-aac --enable-libwebp --disable-openssl --disable-alsa --disable-libpulse --disable-vaapi --disable-vdpau --disable-libxcb --disable-libxcb-shm --disable-libxcb-shape --disable-libxcb-xfixes --disable-appkit --disable-avfoundation --disable-coreimage --disable-audiotoolbox --disable-videotoolbox --disable-securetransport --disable-cuda --disable-cuvid --enable-nonfree --enable-gpl --disable-everything --enable-encoder=gif --enable-filter=concat --disable-vulkan --arch=x86_64 --disable-programs --cc=gcc --cxx=g++ '--extra-cflags=-m64 -fPIC -O3' --extra-ldflags=-m64 
ffmpeg/5.1: RUN: "/home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/configure" --disable-shared --enable-static --prefix=/ '--bindir=${prefix}/bin' '--libdir=${prefix}/lib' --pkg-config-flags=--static --disable-doc --disable-cross-compile --enable-asm --enable-pic --enable-avdevice --enable-avcodec --enable-avformat --enable-swresample --enable-swscale --enable-postproc --enable-avfilter --enable-bzlib --enable-zlib --enable-lzma --enable-iconv --enable-libopenjpeg --enable-libopenh264 --enable-libvorbis --enable-libopus --disable-libzmq --disable-sdl2 --enable-libx264 --enable-libx265 --enable-libvpx --enable-libmp3lame --enable-libfdk-aac --enable-libwebp --disable-openssl --disable-alsa --disable-libpulse --disable-vaapi --disable-vdpau --disable-libxcb --disable-libxcb-shm --disable-libxcb-shape --disable-libxcb-xfixes --disable-appkit --disable-avfoundation --disable-coreimage --disable-audiotoolbox --disable-videotoolbox --disable-securetransport --disable-cuda --disable-cuvid --enable-nonfree --enable-gpl --disable-everything --enable-encoder=gif --enable-filter=concat --disable-vulkan --arch=x86_64 --disable-programs --cc=gcc --cxx=g++ '--extra-cflags=-m64 -fPIC -O3' --extra-ldflags=-m64 
install prefix            /
source path               /home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src
C compiler                gcc
C library                 glibc
ARCH                      x86 (generic)
big-endian                no
runtime cpu detection     yes
standalone assembly       yes
x86 assembler             nasm
MMX enabled               yes
MMXEXT enabled            yes
3DNow! enabled            yes
3DNow! extended enabled   yes
SSE enabled               yes
SSSE3 enabled             yes
AESNI enabled             yes
AVX enabled               yes
AVX2 enabled              yes
AVX-512 enabled           yes
AVX-512ICL enabled        yes
XOP enabled               yes
FMA3 enabled              yes
FMA4 enabled              yes
i686 features enabled     yes
CMOV is fast              yes
EBX available             yes
EBP available             yes
debug symbols             yes
strip symbols             yes
optimize for size         no
optimizations             yes
static                    yes
shared                    no
postprocessing support    yes
network support           yes
threading support         pthreads
safe bitstream reader     yes
texi2html enabled         no
perl enabled              yes
pod2man enabled           yes
makeinfo enabled          yes
makeinfo supports HTML    yes
xmllint enabled           yes

External libraries:
bzlib                   libopus                 lzma
iconv                   libvorbis               sndio
libfdk_aac              libvpx                  xlib
libmp3lame              libwebp                 zlib
libopenh264             libx264
libopenjpeg             libx265

External libraries providing hardware acceleration:
cuda_llvm               v4l2_m2m

Libraries:
avcodec                 avformat                swresample
avdevice                avutil                  swscale
avfilter                postproc

Programs:

Enabled decoders:

Enabled encoders:
gif

Enabled hwaccels:

Enabled parsers:

Enabled demuxers:

Enabled muxers:

Enabled protocols:

Enabled filters:
concat

Enabled bsfs:

Enabled indevs:

Enabled outdevs:

License: nonfree and unredistributable

ffmpeg/5.1: RUN: make -j16
GEN	libavdevice/libavdevice.version
GEN	libavfilter/libavfilter.version
GEN	libavformat/libavformat.version
GEN	libavcodec/libavcodec.version
GEN	libavutil/libavutil.version
GEN	libpostproc/libpostproc.version
GEN	libswresample/libswresample.version
GEN	libswscale/libswscale.version
CC	libavdevice/alldevices.o
CC	libavdevice/avdevice.o
CC	libavdevice/utils.o
CC	libavdevice/version.o
GEN	libavdevice/libavdevice.pc
CC	libavfilter/allfilters.o
CC	libavfilter/audio.o
CC	libavfilter/avf_concat.o
CC	libavfilter/avfilter.o
CC	libavfilter/avfiltergraph.o
CC	libavfilter/buffersink.o
CC	libavfilter/buffersrc.o
CC	libavfilter/colorspace.o
CC	libavfilter/drawutils.o
CC	libavfilter/fifo.o
CC	libavfilter/formats.o
CC	libavfilter/framepool.o
CC	libavfilter/framequeue.o
CC	libavfilter/graphdump.o
CC	libavfilter/graphparser.o
CC	libavfilter/pthread.o
CC	libavfilter/version.o
CC	libavfilter/video.o
GEN	libavfilter/libavfilter.pc
CC	libavformat/allformats.o
CC	libavformat/avformat.o
CC	libavformat/avio.o
CC	libavformat/aviobuf.o
CC	libavformat/demux.o
CC	libavformat/demux_utils.o
CC	libavformat/dump.o
CC	libavformat/format.o
CC	libavformat/id3v1.o
CC	libavformat/id3v2.o
CC	libavformat/isom_tags.o
CC	libavformat/metadata.o
CC	libavformat/mux.o
CC	libavformat/mux_utils.o
CC	libavformat/network.o
CC	libavformat/options.o
CC	libavformat/os_support.o
CC	libavformat/protocols.o
CC	libavformat/riff.o
CC	libavformat/sdp.o
CC	libavformat/url.o
CC	libavformat/utils.o
CC	libavformat/seek.o
CC	libavformat/version.o
GEN	libavformat/libavformat.pc
CC	libavcodec/to_upper4.o
GEN	libavcodec/libavcodec.pc
CC	libpostproc/version.o
CC	libpostproc/postprocess.o
GEN	libpostproc/libpostproc.pc
GEN	libswresample/libswresample.pc
GEN	libswscale/libswscale.pc
GEN	libavutil/libavutil.pc
CC	libavcodec/adts_parser.o
AR	libavdevice/libavdevice.a
CC	libavcodec/allcodecs.o
CC	libavcodec/ac3_parser.o
CC	libavcodec/avcodec.o
CC	libavcodec/avdct.o
CC	libavcodec/avpacket.o
CC	libavcodec/bitstream.o
CC	libavcodec/bitstream_filters.o
CC	libavcodec/bsf.o
CC	libavcodec/codec_desc.o
CC	libavcodec/codec_par.o
CC	libavcodec/d3d11va.o
CC	libavcodec/decode.o
CC	libavcodec/dirac.o
CC	libavcodec/dv_profile.o
CC	libavcodec/encode.o
CC	libavcodec/faandct.o
CC	libavcodec/faanidct.o
CC	libavcodec/fdctdsp.o
CC	libavcodec/frame_thread_encoder.o
CC	libavcodec/get_buffer.o
CC	libavcodec/gif.o
CC	libavcodec/idctdsp.o
CC	libavcodec/imgconvert.o
CC	libavcodec/jfdctfst.o
CC	libavcodec/jfdctint.o
CC	libavcodec/jni.o
CC	libavcodec/jrevdct.o
CC	libavcodec/lzwenc.o
CC	libavcodec/mathtables.o
CC	libavcodec/mediacodec.o
CC	libavcodec/mpeg12framerate.o
CC	libavcodec/parser.o
CC	libavcodec/options.o
CC	libavcodec/parsers.o
src/libavcodec/jrevdct.c:215:28: warning: argument 1 of type ‘int16_t[64]’ {aka ‘short int[64]’} with mismatched bound [-Warray-parameter=]
  215 | void ff_j_rev_dct(DCTBLOCK data)
      |                   ~~~~~~~~~^~~~
In file included from src/libavcodec/jrevdct.c:70:
src/libavcodec/dct.h:62:28: note: previously declared as ‘int16_t *’ {aka ‘short int *’}
   62 | void ff_j_rev_dct(int16_t *data);
      |                   ~~~~~~~~~^~~~
src/libavcodec/jrevdct.c:953:29: warning: argument 1 of type ‘int16_t[64]’ {aka ‘short int[64]’} with mismatched bound [-Warray-parameter=]
  953 | void ff_j_rev_dct4(DCTBLOCK data)
      |                    ~~~~~~~~~^~~~
src/libavcodec/dct.h:63:29: note: previously declared as ‘int16_t *’ {aka ‘short int *’}
   63 | void ff_j_rev_dct4(int16_t *data);
      |                    ~~~~~~~~~^~~~
src/libavcodec/jrevdct.c:1140:29: warning: argument 1 of type ‘int16_t[64]’ {aka ‘short int[64]’} with mismatched bound [-Warray-parameter=]
 1140 | void ff_j_rev_dct2(DCTBLOCK data){
      |                    ~~~~~~~~~^~~~
src/libavcodec/dct.h:64:29: note: previously declared as ‘int16_t *’ {aka ‘short int *’}
   64 | void ff_j_rev_dct2(int16_t *data);
      |                    ~~~~~~~~~^~~~
src/libavcodec/jrevdct.c:1155:29: warning: argument 1 of type ‘int16_t[64]’ {aka ‘short int[64]’} with mismatched bound [-Warray-parameter=]
 1155 | void ff_j_rev_dct1(DCTBLOCK data){
      |                    ~~~~~~~~~^~~~
src/libavcodec/dct.h:65:29: note: previously declared as ‘int16_t *’ {aka ‘short int *’}
   65 | void ff_j_rev_dct1(int16_t *data);
      |                    ~~~~~~~~~^~~~
CC	libavcodec/profiles.o
CC	libavcodec/pthread.o
CC	libavcodec/pthread_frame.o
CC	libavcodec/qsv_api.o
CC	libavcodec/pthread_slice.o
CC	libavcodec/raw.o
CC	libavcodec/simple_idct.o
CC	libavcodec/utils.o
CC	libavcodec/v4l2_buffers.o
CC	libavcodec/v4l2_context.o
CC	libavcodec/v4l2_fmt.o
CC	libavcodec/v4l2_m2m.o
CC	libavcodec/vlc.o
CC	libavcodec/vorbis_parser.o
CC	libavcodec/version.o
CC	libavcodec/x86/constants.o
CC	libavcodec/x86/fdct.o
CC	libavcodec/x86/fdctdsp_init.o
X86ASM	libavcodec/x86/idctdsp.o
CC	libavcodec/x86/idctdsp_init.o
X86ASM	libavcodec/x86/simple_idct.o
X86ASM	libavcodec/x86/simple_idct10.o
CC	libavcodec/xiph.o
CC	libswresample/audioconvert.o
STRIP	libavcodec/x86/simple_idct.o
STRIP	libavcodec/x86/idctdsp.o
CC	libswresample/dither.o
CC	libswresample/rematrix.o
CC	libswresample/options.o
CC	libswresample/resample.o
CC	libswresample/resample_dsp.o
CC	libswresample/swresample.o
CC	libswresample/swresample_frame.o
CC	libswresample/version.o
X86ASM	libswresample/x86/audio_convert.o
CC	libswresample/x86/audio_convert_init.o
X86ASM	libswresample/x86/rematrix.o
CC	libswresample/x86/rematrix_init.o
X86ASM	libswresample/x86/resample.o
CC	libswresample/x86/resample_init.o
CC	libswscale/alphablend.o
CC	libswscale/gamma.o
CC	libswscale/hscale.o
STRIP	libswresample/x86/rematrix.o
CC	libswscale/hscale_fast_bilinear.o
CC	libswscale/input.o
CC	libswscale/options.o
CC	libswscale/output.o
CC	libswscale/rgb2rgb.o
CC	libswscale/slice.o
CC	libswscale/swscale.o
CC	libswscale/swscale_unscaled.o
CC	libswscale/utils.o
STRIP	libswresample/x86/resample.o
CC	libswscale/version.o
CC	libswscale/vscale.o
CC	libswscale/x86/hscale_fast_bilinear_simd.o
X86ASM	libswscale/x86/input.o
X86ASM	libswscale/x86/output.o
CC	libswscale/x86/rgb2rgb.o
X86ASM	libswscale/x86/rgb_2_rgb.o
X86ASM	libswscale/x86/scale.o
X86ASM	libswscale/x86/scale_avx2.o
STRIP	libswscale/x86/scale_avx2.o
CC	libswscale/x86/swscale.o
CC	libswscale/x86/yuv2rgb.o
STRIP	libswscale/x86/rgb_2_rgb.o
X86ASM	libswscale/x86/yuv2yuvX.o
STRIP	libswscale/x86/yuv2yuvX.o
X86ASM	libswscale/x86/yuv_2_rgb.o
CC	libswscale/yuv2rgb.o
STRIP	libavcodec/x86/simple_idct10.o
CC	libavutil/adler32.o
CC	libavutil/aes.o
CC	libavutil/aes_ctr.o
STRIP	libswresample/x86/audio_convert.o
CC	libavutil/audio_fifo.o
CC	libavutil/avsscanf.o
CC	libavutil/avstring.o
CC	libavutil/base64.o
CC	libavutil/blowfish.o
CC	libavutil/bprint.o
CC	libavutil/buffer.o
CC	libavutil/camellia.o
STRIP	libswscale/x86/yuv_2_rgb.o
CC	libavutil/cast5.o
CC	libavutil/channel_layout.o
CC	libavutil/color_utils.o
CC	libavutil/cpu.o
CC	libavutil/crc.o
CC	libavutil/csp.o
CC	libavutil/des.o
CC	libavutil/detection_bbox.o
CC	libavutil/dict.o
CC	libavutil/display.o
CC	libavutil/dovi_meta.o
CC	libavutil/downmix_info.o
CC	libavutil/encryption_info.o
CC	libavutil/error.o
CC	libavutil/eval.o
CC	libavutil/fifo.o
CC	libavutil/file.o
CC	libavutil/file_open.o
CC	libavutil/film_grain_params.o
CC	libavutil/fixed_dsp.o
CC	libavutil/float_dsp.o
CC	libavutil/hash.o
CC	libavutil/frame.o
CC	libavutil/hdr_dynamic_metadata.o
STRIP	libswscale/x86/scale.o
CC	libavutil/hdr_dynamic_vivid_metadata.o
CC	libavutil/hmac.o
STRIP	libswscale/x86/input.o
CC	libavutil/hwcontext.o
CC	libavutil/hwcontext_stub.o
CC	libavutil/imgutils.o
CC	libavutil/integer.o
CC	libavutil/intmath.o
CC	libavutil/lfg.o
CC	libavutil/lls.o
CC	libavutil/log.o
CC	libavutil/log2_tab.o
CC	libavutil/lzo.o
CC	libavutil/mastering_display_metadata.o
CC	libavutil/mathematics.o
CC	libavutil/md5.o
CC	libavutil/mem.o
CC	libavutil/murmur3.o
CC	libavutil/opt.o
CC	libavutil/parseutils.o
CC	libavutil/pixdesc.o
CC	libavutil/pixelutils.o
CC	libavutil/random_seed.o
CC	libavutil/rational.o
CC	libavutil/rc4.o
CC	libavutil/reverse.o
CC	libavutil/ripemd.o
CC	libavutil/samplefmt.o
CC	libavutil/sha.o
CC	libavutil/sha512.o
CC	libavutil/slicethread.o
CC	libavutil/stereo3d.o
CC	libavutil/spherical.o
CC	libavutil/tea.o
CC	libavutil/threadmessage.o
CC	libavutil/time.o
CC	libavutil/timecode.o
CC	libavutil/tree.o
CC	libavutil/twofish.o
CC	libavutil/tx.o
CC	libavutil/tx_double.o
src/libavutil/timecode.c: In function ‘av_timecode_make_string’:
src/libavutil/timecode.c:123:60: warning: ‘%0*d’ directive output may be truncated writing between 1 and 10 bytes into a region of size between 2 and 14 [-Wformat-truncation=]
  123 |     snprintf(buf, AV_TIMECODE_STR_SIZE, "%s%02d:%02d:%02d%c%0*d",
      |                                                            ^~~~
src/libavutil/timecode.c:123:41: note: directive argument in the range [0, 2147483647]
  123 |     snprintf(buf, AV_TIMECODE_STR_SIZE, "%s%02d:%02d:%02d%c%0*d",
      |                                         ^~~~~~~~~~~~~~~~~~~~~~~~
src/libavutil/timecode.c:123:5: note: ‘snprintf’ output between 11 and 32 bytes into a destination of size 23
  123 |     snprintf(buf, AV_TIMECODE_STR_SIZE, "%s%02d:%02d:%02d%c%0*d",
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  124 |              neg ? "-" : "",
      |              ~~~~~~~~~~~~~~~
  125 |              hh, mm, ss, drop ? ';' : ':', ff_len, ff);
      |              ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
CC	libavutil/tx_float.o
CC	libavutil/tx_int32.o
CC	libavutil/utils.o
CC	libavutil/uuid.o
CC	libavutil/version.o
CC	libavutil/video_enc_params.o
CC	libavutil/x86/cpu.o
X86ASM	libavutil/x86/cpuid.o
X86ASM	libavutil/x86/fixed_dsp.o
STRIP	libavutil/x86/cpuid.o
X86ASM	libavutil/x86/float_dsp.o
CC	libavutil/x86/fixed_dsp_init.o
CC	libavutil/x86/float_dsp_init.o
X86ASM	libavutil/x86/imgutils.o
CC	libavutil/x86/imgutils_init.o
STRIP	libavutil/x86/fixed_dsp.o
X86ASM	libavutil/x86/lls.o
X86ASM	libavutil/x86/tx_float.o
CC	libavutil/x86/lls_init.o
CC	libavutil/x86/tx_float_init.o
STRIP	libavutil/x86/imgutils.o
CC	libavutil/xga_font_data.o
CC	libavutil/xtea.o
AR	libavfilter/libavfilter.a
AR	libavformat/libavformat.a
AR	libavcodec/libavcodec.a
STRIP	libavutil/x86/lls.o
AR	libswresample/libswresample.a
STRIP	libavutil/x86/float_dsp.o
AR	libpostproc/libpostproc.a
STRIP	libswscale/x86/output.o
STRIP	libavutil/x86/tx_float.o
AR	libavutil/libavutil.a
AR	libswscale/libswscale.a

ffmpeg/5.1: Package '6ece4764455be2a75c6c9523ed82c4720829412e' built
ffmpeg/5.1: Build folder /home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/build-release
ffmpeg/5.1: Generating the package
ffmpeg/5.1: Temporary package folder /home/si/.conan2/p/t/ffmpeb1f18a22e1c83/p
ffmpeg/5.1: Calling package()
ffmpeg/5.1: RUN: make install DESTDIR=/home/si/.conan2/p/t/ffmpeb1f18a22e1c83/p -j16
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/avio_list_dir.c
INSTALL	libavdevice/libavdevice.a
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/avio_reading.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/decode_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/decode_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/demuxing_decoding.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/encode_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/encode_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/extract_mvs.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/filter_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/filtering_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/filtering_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/http_multiclient.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/hw_decode.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/metadata.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/muxing.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/qsvdec.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/remuxing.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/resampling_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/scaling_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/transcode_aac.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/transcoding.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/vaapi_encode.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/vaapi_transcode.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/README
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/Makefile
INSTALL	libavfilter/libavfilter.a
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/avio_list_dir.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/avio_reading.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/decode_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/decode_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/demuxing_decoding.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/encode_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/encode_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/extract_mvs.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/filter_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/filtering_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/filtering_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/http_multiclient.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/hw_decode.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/metadata.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/muxing.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/qsvdec.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/remuxing.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/resampling_audio.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/scaling_video.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/transcode_aac.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/transcoding.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/vaapi_encode.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/vaapi_transcode.c
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/README
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/doc/examples/Makefile
INSTALL	libavformat/libavformat.a
INSTALL	libavcodec/libavcodec.a
INSTALL	libpostproc/libpostproc.a
INSTALL	libswresample/libswresample.a
INSTALL	libswscale/libswscale.a
INSTALL	libavutil/libavutil.a
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavdevice/avdevice.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavdevice/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavdevice/version_major.h
INSTALL	libavdevice/libavdevice.pc
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavfilter/avfilter.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavfilter/buffersink.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavfilter/buffersrc.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavfilter/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavfilter/version_major.h
INSTALL	libavfilter/libavfilter.pc
INSTALL	libavformat/libavformat.pc
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavformat/avformat.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavformat/avio.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavformat/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavformat/version_major.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/ac3_parser.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/adts_parser.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/avcodec.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/avdct.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/avfft.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/bsf.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/codec.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/codec_desc.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/codec_id.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/codec_par.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/d3d11va.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/defs.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/dirac.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/dv_profile.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/dxva2.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/jni.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/mediacodec.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/packet.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/qsv.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/vdpau.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/version_major.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/videotoolbox.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/vorbis_parser.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavcodec/xvmc.h
INSTALL	libavcodec/libavcodec.pc
INSTALL	libpostproc/libpostproc.pc
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libpostproc/postprocess.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libpostproc/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libpostproc/version_major.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libswresample/swresample.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libswresample/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libswresample/version_major.h
INSTALL	libswresample/libswresample.pc
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libswscale/swscale.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libswscale/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libswscale/version_major.h
INSTALL	libswscale/libswscale.pc
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/adler32.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/aes.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/aes_ctr.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/attributes.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/audio_fifo.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/avassert.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/avstring.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/avutil.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/base64.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/blowfish.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/bprint.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/bswap.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/buffer.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/cast5.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/camellia.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/channel_layout.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/common.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/cpu.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/crc.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/csp.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/des.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/detection_bbox.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/dict.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/display.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/dovi_meta.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/downmix_info.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/encryption_info.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/error.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/eval.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/fifo.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/file.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/frame.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hash.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hdr_dynamic_metadata.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hdr_dynamic_vivid_metadata.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hmac.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_cuda.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_d3d11va.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_drm.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_dxva2.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_qsv.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_mediacodec.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_opencl.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_vaapi.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_videotoolbox.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_vdpau.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/hwcontext_vulkan.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/imgutils.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/intfloat.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/intreadwrite.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/lfg.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/log.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/lzo.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/macros.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/mathematics.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/mastering_display_metadata.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/md5.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/mem.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/motion_vector.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/murmur3.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/opt.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/parseutils.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/pixdesc.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/pixelutils.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/pixfmt.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/random_seed.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/rc4.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/rational.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/replaygain.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/ripemd.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/samplefmt.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/sha.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/sha512.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/spherical.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/stereo3d.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/threadmessage.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/time.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/timecode.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/timestamp.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/tree.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/twofish.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/uuid.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/version.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/video_enc_params.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/xtea.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/tea.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/tx.h
INSTALL	home/si/.conan2/p/t/ffmpeb1f18a22e1c83/b/src/libavutil/film_grain_params.h
INSTALL	libavutil/avconfig.h
INSTALL	libavutil/ffversion.h
INSTALL	libavutil/libavutil.pc

ffmpeg/5.1: package(): Packaged 1 '.md' file: LICENSE.md
ffmpeg/5.1: package(): Packaged 137 '.h' files
ffmpeg/5.1: package(): Packaged 8 '.a' files
ffmpeg/5.1: Created package revision c06e1870dc713ff541778946302c6481
ffmpeg/5.1: Package '6ece4764455be2a75c6c9523ed82c4720829412e' created
ffmpeg/5.1: Full package reference: ffmpeg/5.1#dff19df371919871b224b2c3a8c279e5:6ece4764455be2a75c6c9523ed82c4720829412e#c06e1870dc713ff541778946302c6481
ffmpeg/5.1: Package folder /home/si/.conan2/p/ffmpe81d2e724be980/p
WARN: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
WARN:     'cpp_info.names' used in: libpng/1.6.39, zlib/1.2.13, brotli/1.0.9, ogg/1.3.5, libiconv/1.17, openjpeg/2.5.0, bzip2/1.0.8, freetype/2.13.0, opus/1.3.1, vorbis/1.3.7, libx264/cci.20220602, libfdk_aac/2.0.2, xz_utils/5.4.0, libwebp/1.3.0
WARN:     'cpp_info.build_modules' used in: openjpeg/2.5.0, bzip2/1.0.8, xz_utils/5.4.0, freetype/2.13.0
WARN:     'env_info' used in: libiconv/1.17, bzip2/1.0.8, yasm/1.3.0, pkgconf/1.9.3, freetype/2.13.0
WARN:     'cpp_info.filenames' used in: libfdk_aac/2.0.2, freetype/2.13.0
WARN:     'user_info' used in: freetype/2.13.0

======== Launching test_package ========

======== Computing dependency graph ========
Graph root
    ffmpeg/5.1 (test package): /home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/conanfile.py
Requirements
    brotli/1.0.9#ffbecf553112bc3e25a08c91c28b8b68 - Cache
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d - Cache
    ffmpeg/5.1#dff19df371919871b224b2c3a8c279e5 - Cache
    freetype/2.13.0#a312c096b314c6cfd6b43e290c88928a - Cache
    libfdk_aac/2.0.2#c9eb99853a2067c754bb1a826044d6eb - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335 - Cache
    libmp3lame/3.100#44b12d19316eb2b223d98d3e75dae438 - Cache
    libpng/1.6.39#a4953955f060811c79abff9fe986b6fe - Cache
    libvpx/1.11.0#6fbd55b629d52b82db5786b0dd84a48b - Cache
    libwebp/1.3.0#d92654dbce1f8cb8ae45d5388f86fcfe - Cache
    libx264/cci.20220602#1847939486c0bfc5141898c469f9fff4 - Cache
    libx265/3.4#fb1af2d1f0f1c8aa25510237f1d86c39 - Cache
    ogg/1.3.5#062626875f5c8c59f069f76f148098ef - Cache
    openh264/2.3.1#e61a2e588611d5f611fdca7cbc4947b2 - Cache
    openjpeg/2.5.0#e3684ea8ab1cfc6707a461b0b2d0bb9c - Cache
    opus/1.3.1#6938146de9da3b6817c85b6d522e8e4d - Cache
    vorbis/1.3.7#d936fd8c850f0c7c4021e24e55e6352c - Cache
    xz_utils/5.4.0#186e94237b66f785e4ea095f2f621a56 - Cache
    zlib/1.2.13#13c96f538b52e1600c40b88994de240f - Cache
Build requirements
    gnu-config/cci.20210814#15c3bf7dfdb743977b84d0321534ad90 - Cache
    meson/1.0.0#15586c0ac6f682805875ef903dbe7ee2 - Cache
    nasm/2.15.05#799d63b1672a337584b09635b0f22fc1 - Cache
    ninja/1.11.1#a2f0b832705907016f336839f96963f8 - Cache
    pkgconf/1.9.3#a920b5c7f8d04f22b9fe03db91a864dd - Cache
    yasm/1.3.0#fb800a15413dca19bfaef9e4b5d50694 - Cache

======== Computing necessary packages ========
Requirements
    brotli/1.0.9#ffbecf553112bc3e25a08c91c28b8b68:18545dd496007772cd15d1f792fd745d3faf459e#4c5b1814d03a5ca2661a94bdcc12b0f8 - Cache
    bzip2/1.0.8#411fc05e80d47a89045edc1ee6f23c1d:4530633a3406a354d9c70a71124250144067de22#75e696190aa466c071d9e211bca442f7 - Cache
    ffmpeg/5.1#dff19df371919871b224b2c3a8c279e5:6ece4764455be2a75c6c9523ed82c4720829412e#c06e1870dc713ff541778946302c6481 - Cache
    freetype/2.13.0#a312c096b314c6cfd6b43e290c88928a:a7bd0acd641d011580e2dd3e4cb52f030ef01db4#49f4dc389f7b2639339c61edcf7019c0 - Cache
    libfdk_aac/2.0.2#c9eb99853a2067c754bb1a826044d6eb:2b3e00e93be912c4468bf5911338440f07c9b5ac#a6b721688ee50656ac543612b44836c8 - Cache
    libiconv/1.17#fa54397801cd96911a8294bc5fc76335:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#a82f7b8bf96b3aa38bdcfdc9ba5d1c75 - Cache
    libmp3lame/3.100#44b12d19316eb2b223d98d3e75dae438:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#39ef5ed10ac7f73501e2cc4e5169e718 - Cache
    libpng/1.6.39#a4953955f060811c79abff9fe986b6fe:f98ded23a1dcda1844efab049c7002f85ec1e8d1#1009087d2848fed733f2ecdabe8a7594 - Cache
    libvpx/1.11.0#6fbd55b629d52b82db5786b0dd84a48b:80f2cbb9829b9c7aaebc9f3dec78fb86ac4ff2f3#9fccc154fa4cf19da88a11a3af9afbdd - Cache
    libwebp/1.3.0#d92654dbce1f8cb8ae45d5388f86fcfe:e45efe1f2529ffe2564429d57eafb384b5ecc715#9d95215a5c91c1c271bcedfa8297acfc - Cache
    libx264/cci.20220602#1847939486c0bfc5141898c469f9fff4:3fb4c14de6c821fa43bf9d290f24719bf569ba18#cb5033d96089e1ee6460bd095fad9d48 - Cache
    libx265/3.4#fb1af2d1f0f1c8aa25510237f1d86c39:81185bfb3175decd99a7f82a317c0b5e2fc19fda#ff070ebcf5dd30819411d35021dd3b0f - Cache
    ogg/1.3.5#062626875f5c8c59f069f76f148098ef:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#730c080a224a0fd767ed0d6a54d4995b - Cache
    openh264/2.3.1#e61a2e588611d5f611fdca7cbc4947b2:2b3e00e93be912c4468bf5911338440f07c9b5ac#94e64a3dde81fc60af7273e02dcb54dc - Cache
    openjpeg/2.5.0#e3684ea8ab1cfc6707a461b0b2d0bb9c:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#802cbddfb0ab82327ba4c997a0d53b69 - Cache
    opus/1.3.1#6938146de9da3b6817c85b6d522e8e4d:5cd6a71e35ad633a50574774f9c05abbde0f0b34#e12e619c32edbcb2962fc0149b7acad0 - Cache
    vorbis/1.3.7#d936fd8c850f0c7c4021e24e55e6352c:15c679a1f3354e98b9583bcadddca4d34ae2f0b9#000c8f11c485fc0d0e6296308bb629cb - Cache
    xz_utils/5.4.0#186e94237b66f785e4ea095f2f621a56:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#c54662157e9d9abdcbd569f25c130c8e - Cache
    zlib/1.2.13#13c96f538b52e1600c40b88994de240f:5bc851010eb7b707e5cb2e24cb8ccf0f27989fa9#c0a86acefcee55d678689521d3ea23ea - Cache
Build requirements
    gnu-config/cci.20210814#15c3bf7dfdb743977b84d0321534ad90:da39a3ee5e6b4b0d3255bfef95601890afd80709#22618e30bd9e326eb95e824dc90cc860 - Skip
    meson/1.0.0#15586c0ac6f682805875ef903dbe7ee2:da39a3ee5e6b4b0d3255bfef95601890afd80709#5c8fd51fc33f12e26519674d99afd0e5 - Skip
    nasm/2.15.05#799d63b1672a337584b09635b0f22fc1:3593751651824fb813502c69c971267624ced41a#6fea6fbfb5f1ce0afa9745dfce12485d - Skip
    ninja/1.11.1#a2f0b832705907016f336839f96963f8:3593751651824fb813502c69c971267624ced41a#a53f170b60a46aef75ead8658bdeae05 - Skip
    pkgconf/1.9.3#a920b5c7f8d04f22b9fe03db91a864dd:c0b621fd4b3199fe05075171573398833dba85f4#a231c33e360d8be4e90467b36b88f027 - Skip
    yasm/1.3.0#fb800a15413dca19bfaef9e4b5d50694:3593751651824fb813502c69c971267624ced41a#de209941eeb4029e79a7a33bfefe9a98 - Skip

======== Installing packages ========
brotli/1.0.9: Already installed! (1 of 19)
bzip2/1.0.8: Already installed! (2 of 19)
libfdk_aac/2.0.2: Already installed! (3 of 19)
libiconv/1.17: Already installed! (4 of 19)
libwebp/1.3.0: Already installed! (5 of 19)
ogg/1.3.5: Already installed! (6 of 19)
openjpeg/2.5.0: Already installed! (7 of 19)
opus/1.3.1: Already installed! (8 of 19)
xz_utils/5.4.0: Already installed! (9 of 19)
zlib/1.2.13: Already installed! (10 of 19)
libmp3lame/3.100: Already installed! (11 of 19)
libvpx/1.11.0: Already installed! (12 of 19)
libx264/cci.20220602: Already installed! (13 of 19)
libx265/3.4: Already installed! (14 of 19)
openh264/2.3.1: Already installed! (15 of 19)
libpng/1.6.39: Already installed! (16 of 19)
vorbis/1.3.7: Already installed! (17 of 19)
freetype/2.13.0: Already installed! (18 of 19)
ffmpeg/5.1: Already installed! (19 of 19)
WARN: Usage of deprecated Conan 1.X features that will be removed in Conan 2.X:
WARN:     'cpp_info.names' used in: libpng/1.6.39, zlib/1.2.13, brotli/1.0.9, ogg/1.3.5, libiconv/1.17, openjpeg/2.5.0, bzip2/1.0.8, freetype/2.13.0, opus/1.3.1, vorbis/1.3.7, libx264/cci.20220602, libfdk_aac/2.0.2, xz_utils/5.4.0, libwebp/1.3.0
WARN:     'cpp_info.build_modules' used in: openjpeg/2.5.0, bzip2/1.0.8, xz_utils/5.4.0, freetype/2.13.0
WARN:     'env_info' used in: libiconv/1.17, bzip2/1.0.8, freetype/2.13.0
WARN:     'cpp_info.filenames' used in: libfdk_aac/2.0.2, freetype/2.13.0
WARN:     'user_info' used in: freetype/2.13.0

======== Testing the package ========
Removing previously existing 'test_package' build folder: /home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/build/gcc-12-x86_64-gnu17-release
ffmpeg/5.1 (test package): Test package build: build/gcc-12-x86_64-gnu17-release
ffmpeg/5.1 (test package): Test package build folder: /home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/build/gcc-12-x86_64-gnu17-release
ffmpeg/5.1 (test package): Writing generators to /home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/build/gcc-12-x86_64-gnu17-release/generators
ffmpeg/5.1 (test package): Generator 'CMakeDeps' calling 'generate()'
ffmpeg/5.1 (test package): Generator 'CMakeToolchain' calling 'generate()'
ffmpeg/5.1 (test package): CMakeToolchain generated: conan_toolchain.cmake
ffmpeg/5.1 (test package): CMakeToolchain generated: CMakePresets.json
ffmpeg/5.1 (test package): CMakeToolchain generated: ../../../CMakeUserPresets.json
ffmpeg/5.1 (test package): Generator 'VirtualRunEnv' calling 'generate()'
ffmpeg/5.1 (test package): Generating aggregated env files
ffmpeg/5.1 (test package): Generated aggregated env files: ['conanrun.sh', 'conanbuild.sh']

======== Testing the package: Building ========
ffmpeg/5.1 (test package): Calling build()
ffmpeg/5.1 (test package): Running CMake.configure()
ffmpeg/5.1 (test package): RUN: cmake -G "Unix Makefiles" -DCMAKE_TOOLCHAIN_FILE="/home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/build/gcc-12-x86_64-gnu17-release/generators/conan_toolchain.cmake" -DCMAKE_INSTALL_PREFIX="/home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package" -DCMAKE_POLICY_DEFAULT_CMP0091="NEW" -DCMAKE_BUILD_TYPE="Release" "/home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/."
-- Using Conan toolchain: /home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/build/gcc-12-x86_64-gnu17-release/generators/conan_toolchain.cmake
-- Conan toolchain: C++ Standard 17 with extensions ON
-- The C compiler identification is GNU 12.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Conan: Component target declared 'ffmpeg::avutil'
-- Conan: Component target declared 'ffmpeg::swscale'
-- Conan: Component target declared 'ffmpeg::swresample'
-- Conan: Component target declared 'ffmpeg::postproc'
-- Conan: Component target declared 'ffmpeg::avcodec'
-- Conan: Component target declared 'ffmpeg::avformat'
-- Conan: Component target declared 'ffmpeg::avfilter'
-- Conan: Component target declared 'ffmpeg::avdevice'
-- Conan: Target declared 'ffmpeg::ffmpeg'
-- Conan: Target declared 'LibLZMA::LibLZMA'
-- Conan: Including build module from '/home/si/.conan2/p/xz_ut6d645250c32e1/p/lib/cmake/conan-official-xz_utils-variables.cmake'
-- Conan: Target declared 'Iconv::Iconv'
-- Conan: Target declared 'Freetype::Freetype'
-- Conan: Target declared 'PNG::PNG'
-- Conan: Target declared 'ZLIB::ZLIB'
-- Conan: Target declared 'BZip2::BZip2'
-- Conan: Including build module from '/home/si/.conan2/p/bzip2b9e43c4cfedb4/p/lib/cmake/conan-official-bzip2-variables.cmake'
-- Conan: Component target declared 'brotli::brotlicommon'
-- Conan: Component target declared 'brotli::brotlidec'
-- Conan: Component target declared 'brotli::brotlienc'
-- Conan: Target declared 'brotli::brotli'
-- Conan: Including build module from '/home/si/.conan2/p/freet8ec3fdbd71dd8/p/lib/cmake/conan-official-freetype-variables.cmake'
-- Conan: Target declared 'openjp2'
-- Conan: Target declared 'openh264::openh264'
-- Conan: Component target declared 'Vorbis::vorbis'
-- Conan: Component target declared 'Vorbis::vorbisenc'
-- Conan: Component target declared 'Vorbis::vorbisfile'
-- Conan: Component target declared 'vorbis::vorbisenc-alias'
-- Conan: Component target declared 'vorbis::vorbisfile-alias'
-- Conan: Target declared 'vorbis::vorbis'
-- Conan: Component target declared 'Ogg::ogg'
-- Conan: Component target declared 'Opus::opus'
-- Conan: Target declared 'libx264::libx264'
-- Conan: Target declared 'libx265::libx265'
-- Conan: Target declared 'libvpx::libvpx'
-- Conan: Target declared 'libmp3lame::libmp3lame'
-- Conan: Component target declared 'FDK-AAC::fdk-aac'
-- Conan: Component target declared 'WebP::webpdecoder'
-- Conan: Component target declared 'WebP::sharpyuv'
-- Conan: Component target declared 'WebP::webp'
-- Conan: Component target declared 'WebP::webpdemux'
-- Conan: Component target declared 'WebP::libwebpmux'
-- Conan: Target declared 'libwebp::libwebp'
-- Configuring done
-- Generating done
-- Build files have been written to: /home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/build/gcc-12-x86_64-gnu17-release

ffmpeg/5.1 (test package): Running CMake.build()
ffmpeg/5.1 (test package): RUN: cmake --build "/home/si/Projects/conan-center-index/recipes/ffmpeg/all/test_package/build/gcc-12-x86_64-gnu17-release" -- -j16
[ 50%] Building C object CMakeFiles/test_package.dir/test_package.c.o
[100%] Linking C executable test_package
[100%] Built target test_package


======== Testing the package: Executing test ========
ffmpeg/5.1 (test package): Running test()
ffmpeg/5.1 (test package): RUN: ./test_package
configuration: --disable-shared --enable-static --prefix=/ --bindir='${prefix}/bin' --libdir='${prefix}/lib' --pkg-config-flags=--static --disable-doc --disable-cross-compile --enable-asm --enable-pic --enable-avdevice --enable-avcodec --enable-avformat --enable-swresample --enable-swscale --enable-postproc --enable-avfilter --enable-bzlib --enable-zlib --enable-lzma --enable-iconv --enable-libopenjpeg --enable-libopenh264 --enable-libvorbis --enable-libopus --disable-libzmq --disable-sdl2 --enable-libx264 --enable-libx265 --enable-libvpx --enable-libmp3lame --enable-libfdk-aac --enable-libwebp --disable-openssl --disable-alsa --disable-libpulse --disable-vaapi --disable-vdpau --disable-libxcb --disable-libxcb-shm --disable-libxcb-shape --disable-libxcb-xfixes --disable-appkit --disable-avfoundation --disable-coreimage --disable-audiotoolbox --disable-videotoolbox --disable-securetransport --disable-cuda --disable-cuvid --enable-nonfree --enable-gpl --disable-everything --enable-encoder=gif --enable-filter=concat --disable-vulkan --arch=x86_64 --disable-programs --cc=gcc --cxx=g++ --extra-cflags='-m64 -fPIC -O3' --extra-ldflags=-m64
avcodec version: 59.37.100
avfilter version: 8.44.100
avdevice version: 59.7.100
swresample version: 4.7.100
swscale version: 6.7.100


```

</details>

--- a/recipes/ffmpeg/all/conanfile.py
+++ b/recipes/ffmpeg/all/conanfile.py
@@ -388,6 +388,27 @@ class FFMpegConan(ConanFile):
             return {"cc": "cl.exe", "cxx": "cl.exe"}
         return {}
 
+    def _create_toolchain(self):
+        tc = AutotoolsToolchain(self)
+        # Custom configure script of ffmpeg understands:
+        # --prefix, --bindir, --datadir, --docdir, --incdir, --libdir, --mandir
+        # Options --datadir, --docdir, --incdir, and --mandir are not injected by AutotoolsToolchain  but their default value
+        # in ffmpeg script matches expected conan install layout.
+        # Several options injected by AutotoolsToolchain are unknown from this configure script and must be pruned.
+        # This must be done before modifying tc.configure_args, because update_configre_args currently removes
+        # duplicate configuration keys, even when they have different values, such as list of encoder flags.
+        # See https://github.com/conan-io/conan-center-index/issues/17140 for further information.
+        tc.update_configure_args({
+            "--sbindir": None,
+            "--includedir": None,
+            "--oldincludedir": None,
+            "--datarootdir": None,
+            "--build": None,
+            "--host": None,
+            "--target": None,
+        })
+        return tc
+
     def generate(self):
         env = VirtualBuildEnv(self)
         env.generate()
@@ -402,7 +423,8 @@ class FFMpegConan(ConanFile):
             if v:
                 args.append(f"--disable-{what}")
 
-        tc = AutotoolsToolchain(self)
+        tc = self._create_toolchain()
+
         args = [
             "--pkg-config-flags=--static",
             "--disable-doc",
@@ -593,20 +615,6 @@ class FFMpegConan(ConanFile):
             args.append("--extra-cflags={}".format(" ".join(tc.cflags)))
         if tc.ldflags:
             args.append("--extra-ldflags={}".format(" ".join(tc.ldflags)))
-        # Custom configure script of ffmpeg understands:
-        # --prefix, --bindir, --datadir, --docdir, --incdir, --libdir, --mandir
-        # Options --datadir, --docdir, --incdir, and --mandir are not injected by AutotoolsToolchain  but their default value
-        # in ffmpeg script matches expected conan install layout.
-        # Several options injected by AutotoolsToolchain are unknown from this configure script and must be pruned:
-        tc.update_configure_args({
-            "--sbindir": None,
-            "--includedir": None,
-            "--oldincludedir": None,
-            "--datarootdir": None,
-            "--build": None,
-            "--host": None,
-            "--target": None,
-        })
         tc.configure_args.extend(args)
         tc.generate()
 

--- a/recipes/ffmpeg/all/conanfile.py
+++ b/recipes/ffmpeg/all/conanfile.py
@@ -593,7 +593,6 @@ class FFMpegConan(ConanFile):
             args.append("--extra-cflags={}".format(" ".join(tc.cflags)))
         if tc.ldflags:
             args.append("--extra-ldflags={}".format(" ".join(tc.ldflags)))
-        tc.configure_args.extend(args)
         # Custom configure script of ffmpeg understands:
         # --prefix, --bindir, --datadir, --docdir, --incdir, --libdir, --mandir
         # Options --datadir, --docdir, --incdir, and --mandir are not injected by AutotoolsToolchain  but their default value
@@ -608,6 +607,7 @@ class FFMpegConan(ConanFile):
             "--host": None,
             "--target": None,
         })
+        tc.configure_args.extend(args)
         tc.generate()
 
         if is_msvc(self):

--- a/recipes/ffmpeg/all/conanfile.py
+++ b/recipes/ffmpeg/all/conanfile.py
@@ -261,7 +261,7 @@ class FFMpegConan(ConanFile):
         if self.options.with_bzip2:
             self.requires("bzip2/1.0.8")
         if self.options.with_lzma:
-            self.requires("xz_utils/5.4.0")
+            self.requires("xz_utils/5.4.2")
         if self.options.with_libiconv:
             self.requires("libiconv/1.17")
         if self.options.with_freetype:
@@ -291,7 +291,7 @@ class FFMpegConan(ConanFile):
         if self.options.with_libwebp:
             self.requires("libwebp/1.3.0")
         if self.options.with_ssl == "openssl":
-            self.requires("openssl/1.1.1t")
+            self.requires("openssl/3.1.0")
         if self.options.get_safe("with_libalsa"):
             self.requires("libalsa/1.2.7.2")
         if self.options.get_safe("with_xcb") or self.options.get_safe("with_vaapi"):

